scripts/qemu/configs/default.xml

```diff
@@ -0,0 +1,52 @@
+<!-- SPDX-License-Identifier: BSD-2-Clause
+# Copyright (C) 2024 Intel Corporation.
+-->
+<domain type='qemu'>
+  <name>CascadeLake</name>
```

Comment:
nit: when you'll be adding more configs, pls re-name this one to e.g. "Default"

Reply:
Okay, thanks!